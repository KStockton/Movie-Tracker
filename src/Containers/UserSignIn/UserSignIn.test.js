import React from 'react';
import UserSignin from './UserSignin';
import { shallow } from 'enzyme';

describe('UserSignin', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow (
        <UserSignin />
    )
  })

  it('UserSignin component should match the snapshot', () =>{
    expect(wrapper).toMatchSnapshot()
  })


})