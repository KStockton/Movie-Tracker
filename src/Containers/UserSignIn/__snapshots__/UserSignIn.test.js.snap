// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewUser NewUser component should match the snapshot 1`] = `undefined`;

exports[`UserSignIn UserSignIn component should match the snapshot 1`] = `undefined`;

exports[`UserSignin UserSignIn component should match the snapshot 1`] = `
<SignInUser
  email=""
  handleChange={[Function]}
  handleLogin={[Function]}
  handleToggleForm={[Function]}
  name=""
  newUserBool={false}
  password=""
  status=""
  user={
    Object {
      "favorites": Array [],
    }
  }
/>
`;

exports[`UserSignin UserSignin component should match the snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "clear": [Function],
          "get": [Function],
          "notify": [Function],
          "subscribe": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": [Function],
      },
    }
  }
>
  <ConnectFunction />
</ContextProvider>
`;
